lh-test/src/main/java/io/littlehorse/test/LHClientTestWrapper.java

```diff
@@ -58,17 +60,19 @@ public LHStatus getWfRunStatus(String wfRunId) {
         }
     }
 
+    public VariableValue getVariableValue(String wfRunId, int threadRunNumber, String variableName) {
+        VariableId variableId = VariableId.newBuilder().setName(variableName).setWfRunId(wfRunId).setThreadRunNumber(threadRunNumber).build();
+        return lhClient.getVariable(variableId).getValue();
+    }
+
     public WfRun getWfRun(String wfRunId) {
-        try {
-            return lhClient.getWfRun(WfRunId.newBuilder().setId(wfRunId).build());
-        } catch (StatusRuntimeException e) {
-            throw new LHTestInitializationException(e);
-        }
+        return lhClient.getWfRun(WfRunId.newBuilder().setId(wfRunId).build());
     }
 
     public boolean runWf(WfSpec wfSpec, String wfId, Collection<Arg> args) throws StatusRuntimeException {
         RunWfRequest.Builder req = RunWfRequest.newBuilder()
             .setWfSpecName(wfSpec.getName())
+            .setId(wfId)
```

Comment:
This will throw an exception if `wfId == null`. That's fine if you design it that way